brut.j.dir/src/main/java/brut/directory/DirUtil.java

```diff
@@ -28,6 +28,10 @@
 public class DirUtil {
     private static final Logger LOGGER = Logger.getLogger("");
 
+    private DirUtil(){
```

Comment:
Do you mind explaining what benefit the no-op constructors have? As well as little space here `private DirUtil() {`

Reply:
This is a Java best practice raccomendation
http://www.javapractices.com/topic/TopicAction.do?Id=40

Reply:
I have add a commit with the space.